src/main/flight/pid.c

```diff
@@ -288,7 +288,11 @@ void pidInitConfig(const pidProfile_t *pidProfile)
         Kd[axis] = DTERM_SCALE * pidProfile->pid[axis].D;
     }
     dtermSetpointWeight = pidProfile->dtermSetpointWeight / 127.0f;
-    relaxFactor = 1.0f / (pidProfile->setpointRelaxRatio / 100.0f);
+    if (pidProfile->setpointRelaxRatio == 0) {
+        relaxFactor = 0; //avoid divide by zero
```

Comment:
Indentation here

Reply:
I fixed the other indentation - not sure what to fix here

Reply:
looks good now.